test/models/patient_test.rb

```diff
@@ -27,4 +27,35 @@ class PatientTest < ActiveSupport::TestCase
 
     assert patient.valid?
   end
+
+  test 'is not vaid without medical_history' do
+    patient = Patient.new(
+      name: 'Mike', 
+      birth_date: '13/02/1997', 
+      city: 'Colima', 
+      primary_dx: 'Anything',
+      initial_dx: 'Anything', 
+      final_dx: 'Anything',
+      surgical_history: 'Anything', 
+      treatment: 'Anything'
+    )
```

Comment:
There should be an empty line here
Specs are usually 'divided' in three in blocks: `preparation`, `action` and `expectations`